ipatests/prci_definitions/nightly_ipa-4-9_latest_selinux.yaml

```diff
@@ -588,6 +588,19 @@ jobs:
         timeout: 4800
         topology: *master_1repl_1client
 
+  fedora-latest-ipa-4-9/test_installation_TestInstallwithSHA384withRSA:
+    requires: [fedora-latest-ipa-4-9/build]
+    priority: 50
+    job:
+      class: RunPytest
+      args:
+        build_url: '{fedora-latest-ipa-4-9/build_url}'
+        selinux_enforcing: True
+        test_suite: test_integration/test_installation.py::TestInstallwithSHA384withRSA
+        template: *ci-ipa-4-9-latest
+        timeout: 3600
+        topology: *master_1repl_1client
```

Comment:
`*master_1repl`, no client is needed for the test

Reply:
Fixed it